### PR TITLE
Update npm deps

### PR DESCRIPTION
**Pull request by smorimoto**
Correct unused or unlisted dependencies.

Although it was not intended, this reduces the size of the package 🤷‍♂️

#### Before

```
CJS dist/index.cjs     7.53 KB
ESM dist/index.js     6.82 KB
```

#### After

```
CJS dist/index.cjs     4.94 KB
ESM dist/index.js     4.21 KB
```

### CODE REVIEW

File: package.json
Comment (smorimoto) [inline package.json:42]: Unused dep

File: package.json
Comment (smorimoto) [inline package.json:40]: Unused dep

File: example/package.json
Comment (smorimoto) [inline example/package.json:14]: Unlisted dep

File: package.json
Comment (smorimoto) [inline package.json:49]: Unused dep

**Comment by smorimoto** [comment]
You can confirm this with [Knip](https://knip.dev):

```
npx knip
```

`knip.json`

```json
{
  "$schema": "https://unpkg.com/knip/schema.json",
  "workspaces": {
    ".": {
      "entry": ["src/index.ts", "src/*.test.ts"],
      "project": "src/*.ts"
    },
    "example": {
      "entry": ["index.ts", "*.test.ts"],
      "project": "*.ts"
    }
  }
}
```

File: package.json
Comment (smorimoto) [inline package.json:41]: Unlisted dep

Comment (febuiles) [reply package.json:41]: This is already defined as a transitive dependency for multiple other dependencies in [package-lock.json,](https://github.com/github/dependency-submission-toolkit/blob/5361b6878ba6de3b2a22890462e4a2e7cc08f7db/package-lock.json#L754) why do we need to be explicit here?

**Review by febuiles — APPROVED** [review]
Left a question, but looks good to me!

**Comment by smorimoto** [comment]
Having transitive dependencies always makes the project fragile, so I believe it makes sense to declare them as direct dependencies.

File: package.json
Comment (smorimoto) [reply package.json:41]: Having transitive dependencies always makes the project fragile, so I believe it makes sense to declare them as direct dependencies 🤔

Comment (febuiles) [reply package.json:41]: With Dependabot recently [supporting transitive dependencies](https://github.blog/changelog/2022-09-07-dependabot-unlocks-transitive-dependencies-for-npm-projects/) I don't think this is a problem, but we need to [enable it for NPM](https://github.com/github/dependency-submission-toolkit/blob/main/.github/dependabot.yml)!

The PR was merged after 2.0.2 was cut, thank you again!

Comment (smorimoto) [reply package.json:41]: I will check that! Thanks a lot!

**Comment by smorimoto** [comment]
This change is not included in the latest point release, but are you planning to release this?

**Comment by smorimoto** [comment]
@febuiles Could you publish a new point release for this?

**Comment by febuiles** [comment]
@smorimoto If I push the PR I'll need to get someone's approval, can you update the version and push a PR? I can then be the approver and do the release.

**Comment by smorimoto** [comment]
@febuiles Sure!

**Pull request merged**
